login_server/src/cpp/JSONInterface/JsonUnsecureLogin.cpp

```diff
@@ -158,6 +157,10 @@ Poco::JSON::Object* JsonUnsecureLogin::handle(Poco::Dynamic::Var params)
 		AWAIT(hasElopageTask)
 		result->set("hasElopage", hasElopageTask->hasElopage());
 		return result;
+	case USER_EMAIL_NOT_ACTIVATED:
```

Comment:
The error case must be before the success case to prevent potential problems.
Please move this block above the `... case: USER_COMPLETE` block

Reply:
Technically, I don't see a problem, because only one case can exist at a time. It should therefore not matter in terms of function in which order the cases are checked.
Maybe it is nicer for the eye.
As long as all cases have an return or break at the end. 
USER_NO_GROUP is a special case. Because it is intentional that it hasn't a break after it the success block with USER_NO_PRIVATE_KEY and USER_COMPLETE must follow.

Reply:
For lisibility I agree on the error block before success block!